build/CoreGraphics/lib/CoreGraphicsLib.vcxproj

```diff
@@ -22,7 +22,6 @@
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGBitmapImage.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGColorSpace.mm" />
     <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGContext.mm" />
-    <ClangCompile Include="$(MSBuildThisFileDirectory)..\..\..\Frameworks\CoreGraphics\CGContextCairo.mm" />
```

Comment:
For now, yes. It has unmerged font drawing code.

Keeping the file lets us take merges without conflicts until we're ready to delete it.

---
In reply to: [83311217](https://github.com/Microsoft/WinObjC/pull/1126#discussion_r83311217) [](ancestors = 83311217)